include/wtlgo/Logger.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef Logger_hpp
+#define Logger_hpp
+
+#include <iostream>
+#include <curl/curl.h>
+
+namespace wtlgo {
+
+class ILogger {
+public:
+	virtual ILogger();
+	virtual ~ILogger();
```

Comment:
Please, explicitly define it as _pure virtual_ via 
```suggestion
	virtual ~ILogger() = 0;
```